src/common/constants/authorization/credential.rule.types.constants.ts

```diff
@@ -8,6 +8,8 @@ export const CREDENTIAL_RULE_TYPES_SPACE_AUTHORIZATION_GLOBAL_ADMIN_GRANT =
   'credentialRuleTypes-spaceAuthorizationGlobalAdminGrant';
 export const CREDENTIAL_RULE_TYPES_SPACE_COMMUNITY_APPLY_GLOBAL_REGISTERED =
   'credentialRuleTypes-spaceCommunityApplyGlobalRegistered';
+export const CREDENTIAL_RULE_TYPES_SPACE_STORAGE_FILE_UPLOAD =
```

Comment:
Isn't this CREDENTIAL_RULE and not CREDENTIAL_RULE_TYPES?

Reply:
I changed it to be a rule instead, considering it has the same rule for challenge members.